### ISSUE TITLE
How to deploy?

### ISSUE THREAD

**Issue by quedicesebas** [rustygreen/ng-supabase#2]
I'm facing issues trying to deploy angular 17 with SSR in every cloud provider...

**Comment by lissy93** [comment]
I'd also be interested in hearing Rusty's input on this. 

The work he's done in this repo is awesome, and it's been incredibly useful to me while setting up my new project. I'm able to deploy it ok to Vercel and Netlify, but can only get the client-side stuff working (static SPA). I've spent hours, and not managed to get either SSR or any of the API routes working on Vercel or any other cloud provider.

**Comment by quedicesebas** [comment]
I have been using Firebase App Hosting for a few months now.

**Comment by rustygreen** [comment]
Currently the library doesn't support SSR. I am hosting via static hosting in Cloudflare. This library can be deployed anywhere that's using static hosting (CSR). It shouldn't take a lot to make this compatible with SSR, I just need to find the time to do it. PRs welcome :)

**Comment by quedicesebas** [comment]
But the README syas: "Built on top of [Angular 17](https://blog.angular.io/introducing-angular-v17-4d7033312e4b) with [Signals](https://angular.io/guide/signals) and [SSR](https://angular.io/guide/ssr) support"

**Comment by rustygreen** [comment]
> But the README syas: "Built on top of [Angular 17](https://blog.angular.io/introducing-angular-v17-4d7033312e4b) with [Signals](https://angular.io/guide/signals) and [SSR](https://angular.io/guide/ssr) support"

Good point. It did support it originally, when I started the project but ran into some issues and forgot to remove that. I'll remove it temporarily, until support is added.
